sentry-core/src/main/java/io/sentry/core/SentryOptions.java

```diff
@@ -996,6 +1001,19 @@ public void setSdkVersion(final @Nullable SdkVersion sdkVersion) {
     this.sdkVersion = sdkVersion;
   }
 
+  public boolean isSendDefaultPii() {
+    return sendDefaultPii;
+  }
+
+  public void setSendDefaultPii(boolean sendDefaultPii) {
+    this.sendDefaultPii = sendDefaultPii;
+  }
+
+  @NotNull
+  public EventProcessor getPiiEventProcessor() {
+    return piiEventProcessor;
+  }
+
```

Comment:
I believe it's safer to keep this API out and simply have the options reference where needed to inspect for `sendDefaultPii` before adding any PII in the first place.

This will help with auditing the code for PII, avoid issues where we somehow don't run the processor accidentally (change of order or some other bug) and also forget to add a clean up logic for something we collect as PII without the guard